CMakeLists.txt

```diff
@@ -384,7 +384,7 @@ add_dependencies(moduleVersion opmsimulators)
 set(FLOW_MODELS blackoil brine energy extbo foam gasoil gaswater
                 oilwater oilwater_brine oilwater_polymer
                 oilwater_polymer_injectivity micp polymer solvent)
-set(FLOW_VARIANT_MODELS brine_energy onephase onephase_energy)
+set(FLOW_VARIANT_MODELS brine_energy onephase onephase_energy gasoil_energy)
```

Comment:
This is for variants that do not go into the main flow binary, it seems this one does so it should be in FLOW_MODELS